plotjuggler_plugins/DataStreamZMQ/CMakeLists.txt

```diff
@@ -10,24 +10,24 @@ endif()
 find_package(ZeroMQ QUIET)
 
 if(ZeroMQ_FOUND)
-    message(STATUS "[ZeroMQ] found")
+    # message(STATUS "[ZeroMQ] found")
```

Comment:
can you please cleanup comments?

Reply:
what is going on here?

Are you intentionally removing ZeroMQ plugin?